PowerShellTools/DebugEngine/ScriptDebugger.cs

```diff
@@ -440,6 +440,7 @@ public void Execute(ScriptProgramNode node)
                 if (node.IsFile)
                 {
                     commandLine = String.Format(DebugEngineConstants.ExecutionCommandFormat, node.FileName, node.Arguments);

```

Comment:
string.Format("{0}{1}{2}", GetPrompt(), node.FileName, Environment.NewLine) seems better to me.